etc/module.xml

```diff
@@ -5,5 +5,9 @@
 
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
-    <module name="Aligent_PrerenderIo"/>
+    <module name="Aligent_PrerenderIo">
+        <sequence>
+            <module name="Magento_CatalogStaging"/>
```

Comment:
Not sure how it was missed previously, but `Magento_Catalog` should be here as a bare minimum. Possibly others missing, also